### ISSUE TITLE
Dark Mode

### ISSUE THREAD

**Issue by xlbrto** [drunlin/webappbox#1]
Just a suggestion, it would be pretty helpful if you could implement a dark mode for the main screen of the app and settings. AMOLED would be perfect in my opinion. If the dark theme could be implemented on the webapps as well that would be even better.

**Comment by drunlin** [comment]
@albertocoronado 
Yes. Dark mode is popular. I plan to redesign this app. It will support dark mode.

**Comment by FanboyStudios** [comment]
Would be nice to see a dark mode.

Just curious, are you still working on this app?

Seems like a nice app.
